shenyu-plugin/shenyu-plugin-modify-response/src/main/java/org/apache/shenyu/plugin/modify/response/ModifyResponsePlugin.java

```diff
@@ -175,14 +175,6 @@ public Boolean skip(final ServerWebExchange exchange) {
         private String operation(final String jsonValue, final ModifyResponseRuleHandle handle) {
             DocumentContext context = JsonPath.parse(jsonValue);
             operation(context, handle);
-            if (!CollectionUtils.isEmpty(handle.getReplaceBodyKeys())) {
-                handle.getReplaceBodyKeys().forEach(info -> {
-                    context.renameKey(info.getPath(), info.getKey(), info.getValue());
-                });
-            }
-            if (!CollectionUtils.isEmpty(handle.getRemoveBodyKeys())) {
-                handle.getRemoveBodyKeys().forEach(context::delete);
-            }
             return context.jsonString();
         }
```

Comment:
How about remove `private void operation(final DocumentContext context, final ModifyResponseRuleHandle handle)` and refactor like this?
```
private String operation(final String jsonValue, final ModifyResponseRuleHandle handle) {
            DocumentContext context = JsonPath.parse(jsonValue);
            if (!CollectionUtils.isEmpty(handle.getAddBodyKeys())) {
                handle.getAddBodyKeys().forEach(info -> context.put(info.getPath(), info.getKey(), info.getValue()));
            }
            if (!CollectionUtils.isEmpty(handle.getReplaceBodyKeys())) {
                handle.getReplaceBodyKeys().forEach(info -> context.renameKey(info.getPath(), info.getKey(), info.getValue()));
            }
            if (!CollectionUtils.isEmpty(handle.getRemoveBodyKeys())) {
                handle.getRemoveBodyKeys().forEach(context::delete);
            }
            return context.jsonString();
        }
```

Reply:
Idea prompted me. I thought the line was too long after I changed it, so I changed it back. 😅 (changed)

Reply:
IMO, You don't need to overload one more method `private void operation(final DocumentContext context, final ModifyResponseRuleHandle handle)`.

Reply:
Does that mean just like it used to be? Then I'll turn it off.
> 意思是就像原来那样吗？那我关了(是我看到这段觉得别扭所以给他整一起了)